docs/server/source/server-reference/configuration.md

```diff
@@ -492,3 +493,28 @@ logging of the `core.py` module to be more verbose, you would set the
 ```
 
 **Defaults to**: `"{}"`
+
+
+## Graphite
```

Comment:
Can you change the structure of this subsection to be parallel to the subsection about wsserver.host and wsserver.port ?

e.g.
* It should all be under one heading of the form `## graphite.host`
* Start the JSON blocks with three backticks followed by `js`
* Have an example using the environment variable
* ... Basically follow the exact same structure as the subsection about wsserver.host and wsserver.port. You could copy and paste that subsection here and then just edit it.